### DISCUSSION TITLE
Is there a way to run adetailer before hires?

### DISCUSSION THREAD

**Discussion by StanleyPain** [Bing-su/adetailer#634] [Q&A]
Wondering if there's a way to have adetailer run first, before hires fix.

**Comment by changbowen** [comment]
There is a previous issue #166 but I really want to see this option too... hires fix in txt2img usually gives better results than upscaling in img2img, in my experience

**Comment by jthaloreach** [comment]
Run txt2img with adetailer enabled and hires disabled.  After generation is complete, select the image you wan to run hires on, disable or leave adetailer enabled (depending on your preference) and click this button:
![image](https://github.com/user-attachments/assets/300a639c-d348-4c2f-8cdb-f9b909a4ad33)
